spec/reckon/money_column_spec.rb

```diff
@@ -35,11 +35,23 @@
       Reckon::MoneyColumn.new( ["1.00", ""] ).merge!( 
         Reckon::MoneyColumn.new( ["", "-2.00"] ) ).should == [ 
           Reckon::Money.new( 1.00 ), Reckon::Money.new( -2.00 ) ]
-    end
+      Reckon::MoneyColumn.new( ["1.00", "0"] ).merge!( 
+        Reckon::MoneyColumn.new( ["0", "-2.00"] ) ).should == [ 
+          Reckon::Money.new( 1.00 ), Reckon::Money.new( -2.00 ) ]
+      Reckon::MoneyColumn.new( ["AB1.00C", ""] ).merge!( 
+        Reckon::MoneyColumn.new( ["", "AB-2.00C"] ) ).should == [ 
+          Reckon::Money.new( 1.00 ), Reckon::Money.new( -2.00 ) ]
+      Reckon::MoneyColumn.new( ["AB1.00C", "AB0C"] ).merge!( 
+        Reckon::MoneyColumn.new( ["AB0C", "AB-2.00C"] ) ).should == [ 
```

Comment:
I have no real example, but thought it was nice too allow arbitrary postfix/prefix with numbers in them. In many ways this is more for the columns that are likely to not be money, but something like  account numbers. It will still parse them as money and give them the benefit of the doubt.

I've updated the description of this PR to explain why the suntrust csv didn't work.